File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportJava/Networking/src/main/java/transport/networking/dto/SearchDTO.java
package transport.networking.dto;

import java.io.Serializable;

public class SearchDTO implements Serializable {
    private String destination;
    private String date;
    private String hour;

    public SearchDTO(String destination, String date, String hour) {
        this.destination = destination;
        this.date = date;
        this.hour = hour;
    }

    public String getDestination() {
        return destination;
    }

    public String getDate() {
        return date;
    }

    public String getHour() {
        return hour;
    }
}

File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Network/dto/SearchDTO.cs
using System.Text.Json.Serialization;

namespace FirmaTransportC_.Networking.dto
{
    [Serializable]
    public class SearchDTO
    {
        public string Destination { get; set; }
        public string Date { get; set; }
        public string Hour { get; set; }

        public SearchDTO(string destination, string date, string hour)
        {
            Destination = destination;
            Date = date;
            Hour = hour;
        }

        public SearchDTO()
        {
        }
    }
}
